conf/systemd/plackup_ledgersmb.service

```diff
@@ -0,0 +1,33 @@
+# Place this in /etc/systemd/system/plackup_ledgersmb.service; enable & start
+[Unit]
+Description=LedgerSMB Plack
+After=network.target
+
+[Service]
+Restart=always
+User=www-data
+WorkingDirectory=WORKING_DIR
+#
+ExecStart=/usr/bin/plackup                      \
+    --host ubuntu-vm                            \
+    --port 5000                                 \
+    --daemonize                                 \
+    --workers 5                                 \
+    --access-log /var/log/plackup_ledgersmb.log \
+    -I lib                                      \
+    -E development                              \
+    --server HTTP::Server::PSGI                 \
```

Comment:
This line and the next have development settings. I'm not too fond of that, because many people will simply copy things into place and not really look at what's been put inplace; can we put production config here by default?